include/ua_server.h

```diff
@@ -1056,6 +1056,30 @@ UA_Server_deleteReference(UA_Server *server, const UA_NodeId sourceNodeId,
                           const UA_ExpandedNodeId targetNodeId,
                           UA_Boolean deleteBidirectional);
 
+/**
+ * .. _events:
+ *
+ * Events
+ * ------
+ * The method ``UA_Server_createEvent`` creates an event and represents it as node. The node receives a unique `EventId`
+ * which is automatically added to the node.
+ * The method returns a `NodeId` to the object node which represents the event. The `NodeId` can be used to set the
+ * attributes of the event.
+ *
+ * The method ``UA_Server_triggerEvent`` "triggers" an event by adding it to all monitored items of the specified
+ * origin node and those of all its parents. Any filters specified by the monitored items are automatically applied.
+ * Using this method deletes the node generated by ``UA_Server_createEvent``.
+ */
+#ifdef UA_ENABLE_SUBSCRIPTIONS_EVENTS
+
+UA_StatusCode UA_Server_createEvent(UA_Server *server, const UA_NodeId eventType,
+                                    UA_NodeId *outNodeId);
+
```

Comment:
Event can be

const UA_NodeId eventId

Const, no pointer.

What does outId do???

Reply:
Currently nothing. But the specification requires the user to make all changes to an event by it's EventId. So it's in case something in the future should need it.

Reply:
`outNodeId` is the node ID of the EventType instance (=Object Node) we are creating in `UA_Server_createEvent`.

As mentioned in the corresponding doc of the header:

> The method returns a `NodeId` to the object node which represents the event. The `NodeId` can be used to set the attributes of the event.

The user needs to be able to overwrite/add additional event variables for his own event type using the node id.

@jpfr Does this answer your question?